src/adapter/activateDaffodilDebug.ts

```diff
@@ -85,6 +85,52 @@ async function getFile(fileRequested, label, title) {
   return file
 }
 
+/** Method to select file path to save TDML file
+ * Details:
+ *   Required so that the vscode api commands:
+ *     - extension.dfdl-debug.getValidatedTDMLCopyPath
+ *   can be sent a file instead of always opening up a prompt.
+ */
+async function saveFile(fileRequested, label, title) {
```

Comment:
Concern: this doesn't actually save a file, it returns a path where the data may be saved. Perhaps rename to something like `getTDMLSavePath`?

Reply:
We can use getSavePath function name to make it generic function to get a path to save data, in this case TDML data.

Reply:
Changed the function name to getSavePath